localstack/logging/setup.py

```diff
@@ -78,6 +79,23 @@ def setup_logging_from_config():
         for name, level in trace_internal_log_levels.items():
             logging.getLogger(name).setLevel(level)
 
+    if raw_value := config.LOGGING_OVERRIDE:
```

Comment:
Not sure if we can use this syntax here - this file is imported by the cli if the debug flag is set. Our tests pass, but that flag might break.

Reply:
Good point, thanks!